example/src/TestScreen.tsx

```diff
@@ -107,7 +106,6 @@ export enum TestCategory {
   all = 'all',
   tests = 'tests',
   group = 'group',
-  createdAt = 'createdAt',
```

Comment:
Expected to be deleted?

Reply:
yea, these tests were initially just documenting different behaviors between iOS and Android, but looking at them now, they just pass / fail randomly based off random order that groups are returned. tldr, dont think these tests are useful at the moment, and are just creating extra noise, in my opinion, as the person who wrote them :-P